### PR TITLE
Bounty creation wizard

### PR DESCRIPTION
**Pull request by AmitPaliwal-rs**
Hey, @kevkevinpal I have completed work on ticket #338.

From my side, it is done, but due to new design. Some of the functionalities not working as expected on other pages.

I am creating a draft PR in which the bounty wizard works fine.

In order to make functionalities as expected I will mark this pr for review by tomorrow.

Thanks

### CODE REVIEW

File: frontend/app/src/form/inputs/searchable-select-input.tsx [hunk 1]
```diff
@@ -76,15 +77,22 @@ export default function SearchableSelectInput({
 
   return (
     <>
-      <FieldEnv label={labeltext}>
+      <FieldEnv
+        label={labeltext}
+        isTop={true}
+        style={{
+          border: isBorder ? '1px solid #DDE1E5' : '1px solid #fff'
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/searchable-select-input.tsx:84]: we can use the colors from our color file here

File: frontend/app/src/form/inputs/select-input.tsx [hunk 2]
```diff
@@ -72,3 +79,25 @@ const E = styled.div`
...
+  .euiFormRow_filed {
+    position: relative;
+    .euiFormRow__labelWrapper {
+      margin-bottom: 0px;
+      margin-top: -9px;
+      padding-left: 10px;
+      height: 14px;
+      label {
+        color: #b0b7bc !important;
+        background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/select-input.tsx:93]: same thing with color from colors.tsx

File: frontend/app/src/form/inputs/select-input.tsx [hunk 3]
```diff
@@ -72,3 +79,25 @@ const E = styled.div`
...
+        z-index: 10;
+      }
+    }
+  }
+  .euiFormRow_active {
+    padding: 1px 0;
+    border: 1px solid #82b4ff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/select-input.tsx:100]: colors.tsx

File: frontend/app/src/form/inputs/switch-input.tsx [hunk 4]
```diff
@@ -67,3 +66,20 @@ const ExtraText = styled.div`
...
+  align-items: center;
+  .Label {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 13px;
+    line-height: 35px;
+    display: flex;
+    align-items: center;
+    color: #292c33;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/switch-input.tsx:83]: colors.tsx

File: frontend/app/src/form/inputs/text-area-input.tsx [hunk 5]
```diff
@@ -48,10 +60,40 @@ export default function TextAreaInput({
...
         dangerouslySetInnerHTML={{ __html: extraHTML || '' }}
       />
-    </>
+    </OuterContainer>
   );
 }
 
+const OuterContainer = styled.div`
+  .euiFormRow_active {
+    border: 1px solid #82b4ff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/text-area-input.tsx:69]: colors.tsx

File: frontend/app/src/form/inputs/text-area-input.tsx [hunk 6]
```diff
@@ -48,10 +60,40 @@ export default function TextAreaInput({
...
+    .euiFormRow__labelWrapper {
+      margin-bottom: 0px;
+      margin-top: -9px;
+      padding-left: 10px;
+      height: 14px;
+      label {
+        color: #b0b7bc !important;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/text-area-input.tsx:76]: colors.tsx

File: frontend/app/src/form/inputs/text-area-input.tsx [hunk 7]
```diff
@@ -48,10 +60,40 @@ export default function TextAreaInput({
...
+        background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/text-area-input.tsx:77]: colors.tsx

File: frontend/app/src/form/inputs/text-area-input.tsx [hunk 8]
```diff
@@ -48,10 +60,40 @@ export default function TextAreaInput({
...
+    }
+  }
+  .euiFormRow_filed {
+    .euiFormRow__labelWrapper {
+      margin-bottom: 0px;
+      margin-top: -9px;
+      padding-left: 10px;
+      height: 14px;
+      label {
+        color: #b0b7bc !important;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/text-area-input.tsx:89]: colors.tsx

File: frontend/app/src/form/inputs/text-area-input.tsx [hunk 9]
```diff
@@ -48,10 +60,40 @@ export default function TextAreaInput({
...
+        background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/text-area-input.tsx:90]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+        <label>{label}</label>
+      </div>
+    </TextContainer>
+  );
+}
+
+const ExtraText = styled.div`
+  padding: 0px 10px 5px;
+  margin: -5px 0 10px;
+  color: #b75858;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:44]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+
+const E = styled.div`
+  position: absolute;
+  right: 10px;
+  top: 0px;
+  display: flex;
+  height: 100%;
+  justify-content: center;
+  align-items: center;
+  color: #45b9f6;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:59]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 12]
```diff
@@ -0,0 +1,127 @@
...
+  position: relative;
+`;
+
+const TextContainer = styled.div`
+  margin-bottom: 13px;
+  .label-float {
+    position: relative;
+    padding-top: 13px;
+    font-size: 13px;
+    color: #b0b7bc;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:73]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+    font-family: Barlow;
+  }
+
+  .label-float input {
+    border: 1px solid #dde1e5;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:78]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 14]
```diff
@@ -0,0 +1,127 @@
...
+    font-size: 13px;
+    font-family: Barlow;
+    transition: all 0.1s linear;
+    -webkit-transition: all 0.1s linear;
+    -moz-transition: all 0.1s linear;
+    -webkit-appearance: none;
+  }
+
+  .label-float input:focus {
+    border: 1px solid #82b4ff;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:92]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 15]
```diff
@@ -0,0 +1,127 @@
...
+
+  .label-float label {
+    pointer-events: none;
+    position: absolute;
+    top: calc(50% - 8px);
+    left: 15px;
+    transition: all 0.1s linear;
+    -webkit-transition: all 0.1s linear;
+    -moz-transition: all 0.1s linear;
+    background-color: white;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:107]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 16]
```diff
@@ -0,0 +1,127 @@
...
+    padding: 5px;
+    box-sizing: border-box;
+  }
+
+  .label-float input:required:invalid + label {
+    color: red;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:113]: colors.tsx

File: frontend/app/src/form/inputs/textInputnew.tsx [hunk 17]
```diff
@@ -0,0 +1,127 @@
...
+  }
+  .label-float input:focus:required:invalid {
+    border: 2px solid red;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/textInputnew.tsx:116]: colors.tsx

File: frontend/app/src/form/inputs/widgets/PeopleSearch.tsx [hunk 18]
```diff
@@ -0,0 +1,190 @@
...
+      <input
+        value={searchValue}
+        className="SearchInput"
+        onChange={(e) => {
+          handler(e, '');
+        }}
+        placeholder={'Search'}
+        style={{
+          background: '#fff',
+          color: '#292C33',
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/PeopleSearch.tsx:43]: colors.tsx

File: frontend/app/src/form/inputs/widgets/PeopleSearch.tsx [hunk 19]
```diff
@@ -0,0 +1,190 @@
...
+    <SearchOuterContainer>
+      <input
+        value={searchValue}
+        className="SearchInput"
+        onChange={(e) => {
+          handler(e, '');
+        }}
+        placeholder={'Search'}
+        style={{
+          background: '#fff',
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/PeopleSearch.tsx:42]: colors.tsx

File: frontend/app/src/form/inputs/widgets/PeopleSearch.tsx [hunk 20]
```diff
@@ -0,0 +1,190 @@
...
+      background: #ffffff;
+      border: 1px solid #dde1e5;
+      border-radius: 200px;
+      outline: none;
+    }
+    :active {
+      .SearchText {
+        outline: none;
+        background: #ffffff;
+        border: 1px solid #dde1e5;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/PeopleSearch.tsx:131]: colors.tsx

File: frontend/app/src/form/inputs/widgets/PeopleSearch.tsx [hunk 21]
```diff
@@ -0,0 +1,190 @@
...
+        border-radius: 200px;
+        outline: none;
+      }
+    }
+  }
+  .PeopleList {
+    background: #f2f3f5;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/PeopleSearch.tsx:138]: colors.tsx

File: frontend/app/src/form/inputs/widgets/PeopleSearch.tsx [hunk 22]
```diff
@@ -0,0 +1,190 @@
...
+          align-items: center;
+          object-fit: cover;
+        }
+        .PeopleName {
+          font-family: Barlow;
+          font-style: normal;
+          font-weight: 500;
+          font-size: 13px;
+          line-height: 16px;
+          color: #3c3f41;
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/PeopleSearch.tsx:171]: colors.tsx

File: frontend/app/src/form/inputs/widgets/date.tsx [hunk 23]
```diff
@@ -14,14 +16,31 @@ function Date({ label, value, handleChange }: any) {
...
   );
 }
 export default memo(Date);
+
+interface datePickerProps {
+  border?: boolean;
+}
+
+const DataPicker = styled(EuiDatePicker)<datePickerProps>`
+  border: 1px solid ${(p) => (p.border ? '#82B4FF' : '#DDE1E5')};
```
Comment (kevkevinpal) [inline frontend/app/src/form/inputs/widgets/date.tsx:42]: colors.tsx

File: frontend/app/src/people/personViewSlim.tsx [hunk 24]
```diff
@@ -341,6 +386,37 @@ export default function PersonView(props: any) {
...
+                onClick={() => {
+                  if (ui.meInfo && ui.meInfo?.owner_alias) {
+                    setShowFocusView(true);
+                  } 
+                  // else {
+                  //   showModal();
+                  // }
+                }}
+              />
+          <div>vdsvv</div> */}
```
Comment (kevkevinpal) [inline frontend/app/src/people/personViewSlim.tsx:419]: if we want to remove this we can just delete to keep the code clean, if some work needs to be done to use this lets add a TODO

File: frontend/app/src/people/utils/QR.tsx [hunk 25]
```diff
@@ -55,7 +55,7 @@ export default function QR(props) {
           alignItems: 'center'
         }}
       >
-        <div style={{ background: '#fff', height: 63, width: 63, borderRadius: '50%' }} />
+        <div style={{ background: '#fff', height: 75, width: 75,  }} />
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/QR.tsx:58]: colors.tsx

File: frontend/app/src/people/utils/connectCard.tsx [hunk 26]
```diff
@@ -12,24 +12,28 @@ export default function ConnectCard(props) {
...
-    >
+      // close={(e) => {
+        //   e.stopPropagation();
+        //   props.dismiss();
+        // }}
+        overlayClick={(e: React.SyntheticEvent)=>{
+          props.dismiss();
+        }}
+        visible={visible}>
+      {/* <div style={{position:'relative'}}> */}
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:26]: we can remove this div

File: frontend/app/src/people/utils/connectCard.tsx [hunk 27]
```diff
@@ -12,24 +12,28 @@ export default function ConnectCard(props) {
...
-      close={(e) => {
-        e.stopPropagation();
-        props.dismiss();
-      }}
-      visible={visible}
-    >
+      // close={(e) => {
+        //   e.stopPropagation();
+        //   props.dismiss();
+        // }}
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:21]: if we dont need lets remove

File: frontend/app/src/people/utils/connectCard.tsx [hunk 28]
```diff
@@ -45,11 +49,17 @@ export default function ConnectCard(props) {
...
               height={48}
               width={'100%'}
-            />
+              />
           </a>
         </div>
       </div>
+        <div style={{position:'absolute' , bottom:'-36px',  width: 310, backgroundColor:'transparent', display:'flex', justifyContent:'center'}} >
+          <img src="/static/scan_qr.svg" alt="scan" />
+          <div style={{marginLeft:'12px', color:'#FFFFFF'}} >Scan or paste in Sphinx</div>
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:58]: colors.tsx

File: frontend/app/src/people/utils/connectCard.tsx [hunk 29]
```diff
@@ -45,11 +49,17 @@ export default function ConnectCard(props) {
               imgSize={27}
               height={48}
               width={'100%'}
-            />
+              />
           </a>
         </div>
       </div>
+        <div style={{position:'absolute' , bottom:'-36px',  width: 310, backgroundColor:'transparent', display:'flex', justifyContent:'center'}} >
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:56]: colors.tsx

File: frontend/app/src/people/utils/connectCard.tsx [hunk 30]
```diff
@@ -77,33 +87,32 @@ const W = styled.div`
...
+  font-size: 20px;
+  line-height: 26px;
   /* or 129% */
 
   text-align: center;
 
   /* Main bottom icons */
 
-  color: #5f6368;
+  color: #3c3f41;
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/connectCard.tsx:115]: colors.tsx

File: frontend/app/src/people/utils/noneSpace.tsx [hunk 31]
```diff
@@ -3,6 +3,7 @@ import styled from 'styled-components';
 import { Button } from '../../sphinxUI';
 
 export default function NoneSpace(props) {
+  console.log(props);
```
Comment (kevkevinpal) [inline frontend/app/src/people/utils/noneSpace.tsx:6]: we can remove this log

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 32]
```diff
@@ -575,6 +655,550 @@ export default function WantedSummary(props: any) {
     }
 
     // desktop view
+    if (fromBountyPage) {
+      return (
+        <>
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:660]: do we need this here? Seems like the two things we return are wrapped entirely

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 33]
```diff
@@ -575,6 +655,550 @@ export default function WantedSummary(props: any) {
...
+                  <IconButton
+                    width={220}
+                    height={48}
+                    style={{
+                      position: 'absolute',
+                      marginBottom: '49px',
+                      bottom: '0',
+                      marginLeft: '36px',
+                      background: '#fff',
+                      border: '1px solid #DDE1E5'
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:927]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 34]
```diff
@@ -575,6 +655,550 @@ export default function WantedSummary(props: any) {
...
+                    }}
+                    text={badgeRecipient ? 'Badge Awarded' : 'Award Badge'}
+                    endingImg={'/static/award.svg'}
+                    hoverColor={'#fff'}
+                    activeColor={'#fff'}
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:932]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 35]
```diff
@@ -575,6 +655,550 @@ export default function WantedSummary(props: any) {
...
+                    textStyle={{
+                      width: '130px',
+                      display: 'flex',
+                      justifyContent: 'center',
+                      fontFamily: 'Barlow',
+                      marginLeft: '30px',
+                      color: '#5F6368'
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:939]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 36]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+  display: flex;
+  justify-content: space-between;
+`;
+
+const CreatorDescription = styled.div<styleProps>`
+  min-width: 600px;
+  max-width: 600px;
+  min-height: 768px;
+  border-right: ${(p) => (p?.paid ? '3px solid #86D9B9' : '1px solid #ebedef')};
+  background: #fff;
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1597]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 37]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+const TitleBox = styled.div`
+  // padding-top: 48px;
+  font-family: 'Barlow';
+  font-style: normal;
+  font-weight: 600;
+  font-size: 22px;
+  line-height: 26px;
+  display: flex;
+  align-items: center;
+  color: #292c33;
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1632]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 38]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+  min-height: 548px;
+  max-height: 548px;
+  overflow-y: scroll;
+`;
+
+const AssigneeProfile = styled.div`
+  min-width: 292px;
+  max-width: 292px;
+  min-height: 768px;
+  background: #fff;
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1647]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 39]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+  display: flex;
+  flex-wrap: wrap;
+  width: 80%;
+  margin-top: 16px;
+  margin-bottom: 23.25px;
+`;
+
+const CodingLabels = styled.div<codingLangProps>`
+  padding: 0px 8px;
+  border: ${(p) => (p.border ? p?.border : '1px solid #000')};
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1675]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 40]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+  color: ${(p) => (p.color ? p?.color : '#000')};
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1676]: color.tsx

File: frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx [hunk 41]
```diff
@@ -957,3 +1572,176 @@ const Img = styled.div<ImageProps>`
...
+  background: ${(p) => (p.background ? p?.background : '#fff')};
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummary.tsx:1677]: color.tsx

File: frontend/app/src/sphinxUI/bountyModal_button_set.tsx [hunk 42]
```diff
@@ -0,0 +1,106 @@
...
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+  .buttonImage {
+    filter: brightness(0) saturate(100%) invert(85%) sepia(10%) saturate(180%) hue-rotate(162deg)
+      brightness(87%) contrast(83%);
+  }
+  :hover {
+    border: 1px solid #b0b7bc;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/bountyModal_button_set.tsx:89]: colors.tsx

File: frontend/app/src/sphinxUI/bountyModal_button_set.tsx [hunk 43]
```diff
@@ -0,0 +1,106 @@
...
+  }
+  :active {
+    border: 1px solid #8e969c;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/bountyModal_button_set.tsx:92]: colors.tsx

File: frontend/app/src/sphinxUI/bountyModal_button_set.tsx [hunk 44]
```diff
@@ -0,0 +1,106 @@
...
+        brightness(91%) contrast(86%);
+    }
+  }
+  .ButtonText {
+    font-family: 'Barlow';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: #5f6368;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/bountyModal_button_set.tsx:104]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 45]
```diff
@@ -0,0 +1,147 @@
...
+
+  return (
+    <SearchOuterContainer>
+      <input
+        className="SearchInput"
+        onChange={handler}
+        placeholder={'Search'}
+        style={{
+          background: '#fff',
+          color: '#292C33',
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:32]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 46]
```diff
@@ -0,0 +1,147 @@
...
+  }, [searchValue, props]);
+
+  return (
+    <SearchOuterContainer>
+      <input
+        className="SearchInput"
+        onChange={handler}
+        placeholder={'Search'}
+        style={{
+          background: '#fff',
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:31]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 47]
```diff
@@ -0,0 +1,147 @@
...
+export default AutoComplete;
+
+const SearchOuterContainer = styled.div`
+  min-height: 347px;
+  max-height: 347px;
+  min-width: 336px;
+  max-width: 336px;
+  overflow-x: hidden;
+  overflow-y: scroll;
+  background: #fff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:78]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 48]
```diff
@@ -0,0 +1,147 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 25px 0 0 0;
+
+  .SearchInput {
+    background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:85]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 49]
```diff
@@ -0,0 +1,147 @@
...
+    border: 1px solid #dde1e5;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:86]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 50]
```diff
@@ -0,0 +1,147 @@
...
+    border-radius: 200px;
+    width: 292px;
+    height: 40px;
+    outline: none;
+    overflow: hidden;
+    caret-color: #a3c1ff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:92]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 51]
```diff
@@ -0,0 +1,147 @@
...
+    margin-bottom: 8px;
+    padding: 0px 18px;
+
+    :focus-visible {
+      background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:97]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 52]
```diff
@@ -0,0 +1,147 @@
...
+      border: 1px solid #dde1e5;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:98]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 53]
```diff
@@ -0,0 +1,147 @@
...
+      border-radius: 200px;
+      outline: none;
+    }
+    :active {
+      .SearchText {
+        outline: none;
+        background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:105]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 54]
```diff
@@ -0,0 +1,147 @@
...
+        border: 1px solid #dde1e5;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:106]: colors.tsx

File: frontend/app/src/sphinxUI/custom_autocomplete.tsx [hunk 55]
```diff
@@ -0,0 +1,147 @@
...
+        border-radius: 200px;
+        outline: none;
+      }
+    }
+  }
+  .PeopleList {
+    background: #ffffff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/custom_autocomplete.tsx:113]: colors.tsx

File: frontend/app/src/sphinxUI/modal.tsx [hunk 56]
```diff
@@ -91,6 +111,35 @@ export default function Modal(props: any) {
...
+              />
+            </CircL>
+          </LNew>
+        )}
+        {nextArrowNew && (
+          <RNew>
+            <CircR>
+              <IconButton
+                icon={'chevron_right'}
+                iconStyle={{ color: '#fff' }}
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/modal.tsx:134]: colors.tsx

File: frontend/app/src/sphinxUI/select.tsx [hunk 57]
```diff
@@ -70,6 +77,9 @@ const S = styled(EuiSuperSelect as any)`
     background: #ffffff !important;
     background-color: #ffffff !important;
   }
+  .euiPanel {
+    background: #fff;
```
Comment (kevkevinpal) [inline frontend/app/src/sphinxUI/select.tsx:81]: colors.tsx

**Review by kevkevinpal — COMMENTED** [review]
Added some initial comments alot of them are for adding the corresponding colors.tsx value

**Comment by AmitPaliwal-rs** [comment]
Hi @kevkevinpal, Branch looks good.

**Pull request merged**